plugin.js

```diff
@@ -62,11 +66,15 @@ processor.postprocess = (messages, filename) => {
   const firstMsg = messages[0];
   const isFirstLineOverflown = firstMsg.ruleId === 'max-len' &&
                                firstMsg.line === 1;
-  const isUseStrictAdded = firstMsg.source.startsWith(USE_STRICT) &&
-                           modifiedFiles.has(filename);
-  // TODO(aqrln): respect user's max-len rule settings
-  const inducedOverflow = firstMsg.source.slice(USE_STRICT.length).length <= 80;
-  if (isFirstLineOverflown && isUseStrictAdded && inducedOverflow) {
+  let isPrefixAdded = false;
+  let inducedOverflow = false;
+  if (modifiedFiles.has(filename)) {
+    const prefix = addedPrefixes.get(filename);
+    isPrefixAdded = firstMsg.source.startsWith(prefix);
+    // TODO(aqrln): respect user's max-len rule settings
+    inducedOverflow = firstMsg.source.slice(prefix.length).length <= 80;
```

Comment:
`firstMsg.source.length - prefix.length <= 80`

Reply:
Also, there's another bug: the whole file might comprise a single line, so we need to keep track of added semicolons too.